src/routes/url.ts

```diff
@@ -18,14 +19,15 @@ const saveUrl: Route<{ Body: { url: string } }> = {
 	},
 	async handler(request) {
 		await validateUrl(request.body.url)
-		const url = await this.storage.url.save(request.body.url)
+		const urlRequestData = { url: request.body.url, ip: request.ip } as UrlRequestData
```

Comment:
So we've decided we're not getting the request URL for now, right?

Can you add a ticket for that?

Reply:
Yes. Let's put a ticket for it in the future perhaps.